### ISSUE TITLE
Not asking for permission. Video stream not available

### ISSUE THREAD

**Issue by lolaswift** [AckerApple/ack-angular-webcam#44]
Hi Acker, my code works well on browser but not on mobile devices. On mobile devices, I don't get the permission prompt. I followed your doc. I don't know what went wrong. You test url https://ackerapple.github.io/ack-angular-webcam/ works well on both web and mobile. Is it possible for you to share the source code of it?
thx.

**Comment by AckerApple** [comment]
[The example app is built here](https://github.com/AckerApple/ack-angular-webcam/tree/master/example/app)

The master template that drives the [entire example is here](https://github.com/AckerApple/ack-angular-webcam/blob/master/example/app/app.pug) in pug which is HTML short-hand

The app component that drives the [example is here](https://github.com/AckerApple/ack-angular-webcam/blob/master/example/app/app.component.ts)

Try to absorb all that and get back to me with new questions. Take your time please

**Comment by lolaswift** [comment]
Quick response as always. Appreciated! With those sample code, I should be able to fix my issue. I will report back soon

**Comment by AckerApple** [comment]
I love the little bear icon but I'd like to remember those I work with as faces.

Care to accept my request for a real icon of yourself?

Absolutely feel free to not do so

**Comment by AckerApple** [comment]
Anything to report? I’ll be touching on this package today to fix camera switching on Android for another reported issue

**Comment by lolaswift** [comment]
Hi Acker, I refactored my code by following your doc and your sample code and it is working now. I don't really know know why it made a difference but it's obvious that it's my own implementation issue.
Many thanks again for following up the issue. And glad to know that you'd fixed the android issue which was the issue I was facing too.  ack-angular-webcam is the best among all other packages I've tested. Thumbs up for the features provided and the quality of the code.
